src/PresentationalComponents/Cards/OverviewDashbarCard/__tests__/OverviewDashbarCard.test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import { OverviewDashbarCard } from '../OverviewDashbarCard';
+import {
+  PATHWAYS,
+  INCIDENTS,
+  CRITICAL_RECOMMENDATIONS,
+  IMPORTANT_RECOMMENDATIONS,
+} from '../../../../AppConstants';
+
+// this component returns the appropriate Card component based on the title it receives
+describe('OverviewDashbarCard', () => {
+  const user = userEvent.setup();
+
+  it('Should render Pathways Card', async () => {
+    const onClickFilterByTitle = jest.fn();
+    render(
+      <OverviewDashbarCard
+        title={PATHWAYS}
+        count={1}
+        onClickFilterByTitle={onClickFilterByTitle}
+      />
+    );
+    await waitFor(() => screen.getByText(PATHWAYS));
+
+    // ensure that the correct text is displayed
+    screen.getByText(PATHWAYS);
+    screen.getByText(/1/);
+
+    // ensure filtering is called by clicking the count
+    const filterByTitleBtn = screen.getByTestId(PATHWAYS);
+    await user.click(filterByTitleBtn);
+    expect(onClickFilterByTitle).toHaveBeenCalledWith(PATHWAYS);
```

Comment:
It might be better to put these into two separate tests, one that checks for the test and the other to check the onclick event.

Reply:
Done 👍🏼 
Created one test that renders the component, and another for checking that the filtering function is called (for every single card type that can be rendered